PIM/API/03_KeyConcepts.md

```diff
@@ -0,0 +1,137 @@
+
+Every attribute has a defined data type, which determines the type of value the attribute may contain, for example, a series of characters (data type: string) or a whole number (data type: integer). For detailed information about the data types, see [Data type list](../../DataHub/UserInterface/04_DataTypeList.md).
+
+You can create your own attributes in your instance via user interface, as well as edit or deactivate existing attributes. For detailed information, see [Manage the attributes](../../DataHub/Integration/01_ManageAttributes.md) in the *DataHub* documentation.
+
+
+The following examples are based on predefined *PIM* attributes. If you have created your own attributes in your system, the names and keys may differ.
+
+| Attribute        | Attribute key               | Data type |
+|------------------|-----------------------------|-----------|
+| Name             | *_pim_art_name*             | string    |
+| Price                                             | *_pim_price*                                        | PIM price field   |
+| Tax class                                         | *_taxes_tax_class*                                  | taxclass |
+| Product description                               | *_pim_products_description*                         | textfield    |
+| Product description (multi-scope, multi-language) | *_pim_products_description__scope__language_code*   | textfield    |
+
+
+[comment]: <> (Frage: Tax class can be data type "tax class" and "tree node", i.e. Standard, ermäßigt. Was ist data type taxclass? Wieso ist es nicht zu finden in API products? Owning module ist Taxes? Ähnlich wie pim_price and PIM price field data type. Warum nicht data type number?)
+
+The following graphic visualizes a series of attributes, within an attribute set, maintained in different scopes and languages, and how these attributes define a specific product: 
+
+![Attribute](../../Assets/Screenshots/PIM/API/Attribute.png "[Attribute]")
+
+
+
+## Attribute sets
+
+An attribute set includes a number of attributes that define an entity type, for example, a specific type of product in the case of the *PIM* module. All attribute sets can be created and maintained in the *DataHub* module. Therefore, all attribute sets available in the system are displayed in *DataHub*. Besides, some entity-specific attribute sets, such as a product-related attribute sets or offer-related attribute sets, can be created and maintained in the corresponding module, that is, in *PIM* or *Omni-Channel* respectively. If you install additional plugins to communicate with third-party systems, such as external sales channels or service providers, other attribute sets may be created via driver.
```

Comment:
Vorschlag: Besides, some entity-specific attribute sets, such as a product-related or offer-related attribute set, can be created and maintained  in PIM or Omni-Channel.